crypto/object-hasher/src/index.ts

```diff
@@ -33,3 +36,10 @@ function hashUnknown (object: unknown, options: hash.BaseOptions): string {
 
 export const hashObjectWithoutSorting = (object: unknown): string => hashUnknown(object, withoutSortingOptions)
 export const hashObject = (object: unknown): string => hashUnknown(object, withSortingOptions)
+
+export type PackageExtensionsChecksum = `sha256-${string}` | undefined
+export function createPackageExtensionsChecksum (packageExtensions: Record<string, PackageExtension> | undefined): PackageExtensionsChecksum {
+  if (!packageExtensions || isEmpty(packageExtensions)) return undefined
+  const packageExtensionsChecksum = hash(packageExtensions, withSortingOptions)
+  return `sha256-${packageExtensionsChecksum}`
+}
```

Comment:
This function could be more generic, but the name `hashObject` is already taken. Maybe we should change `hashObject` to return `undefined` for empty objects?